### PR TITLE
Bug(learn) - List item's font size larger than surrounding text

### LINKED ISSUE #40220: Bug(learn) - List item's font size larger than surrounding text
While reviewing some new PRs that used unordered lists, I noticed the font size of the bulleted items was significantly larger than the surround text in a challenge's description or instruction section.  I asked a few other staff if they noticed it, and they agreed something was not right.  Today, I took the time to go back in as far as 12 months in the repo's commits and discovered it has been this way for at least a year (probably much longer).

See screenshot below for the large text for the [Use class Syntax to Define a Constructor FunctionPassed](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function) challenge.

image

### PR DESCRIPTION
**Pull request by prolland006**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40220

I changed the css because the text size was different in <Code>ul</Code>
I tested in local

### CODE REVIEW

**Comment by ieahleen** [comment]
Thank you for your PR, but please go through the checklist, and add the issue number fixed by this PR

**Comment by prolland006** [comment]
#40220 Bug(learn) - List item's font size larger than surrounding text

**Comment by RandellDawson** [comment]
@prolland006 Please read our [contributing documentation](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=proposing-a-pull-request), specifically item number 4.

**Review by moT01 — DISMISSED** [review]
Thanks for the PR @prolland006 🎉 For future reference, it's also best to start your branch with one of things described under [#3 here](https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally). You can just leave this one though.

I'm fine with these changes - this group of styles could possibly go in [the description component css](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/client/src/templates/Challenges/components/challenge-description.css) instead of here. It works either way, but I do think that file is a little more appropriate - so if you want to move them there, you can @prolland006 - or you can just leave it and we'll see what another reviewer thinks.

**Comment by prolland006** [comment]
@moT01 I did your modification : change the place of css.

**Review by moT01 — DISMISSED** [review]
LGTM 🎉

File: client/src/templates/Challenges/components/challenge-description.css [hunk 1]
```diff
@@ -7,3 +7,9 @@
   margin-bottom: 1.45rem;
   font-size: 0.9rem;
 }
+
+#description p,
+#instructions p,
+#description ul {
+  font-family: 'Lato', sans-serif;
+}
```
Comment (nhcarrigan) [inline client/src/templates/Challenges/components/challenge-description.css:15]:
```suggestion
#description p,
#instructions p,
#description ul,
#instructions ul {
  font-family: 'Lato', sans-serif;
}
```
If we are going this route, we need to also set this for the instructions section as well.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Heya @prolland006~!

Thank you for your contribution - I've got one slight change to add.

I'm not sure that changing the font is necessarily the best approach here. The output of the new font is somewhat squished together:
image

**Review by RandellDawson — DISMISSED** [review]
@nhcarrigan Can you change your suggestion to also include `#description ol` and `#instructions ol` to be able to properly display projects like the `Personal Library`, so:

image


becomes:

image

**Comment by nhcarrigan** [comment]
Updated 🙂

**Comment by RandellDawson** [comment]
> Updated

You beat me to it.

**Comment by RandellDawson** [comment]
@prolland006 If you can make the suggested change, we can get this PR merged.  Thanks.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that our workflow requires at least one additional approving review before this can be merged.

**Review by moT01 — APPROVED** [review]
LGTM 🎉 Thanks for fixing this issue @prolland006 and congratulations on your first contribution to this repo 🎉

**Pull request merged**